internal/cli/cmd/tenant/show.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright (c) 2024 OceanBase
+ob-operator is licensed under Mulan PSL v2.
+You can use this software according to the terms and conditions of the Mulan PSL v2.
+You may obtain a copy of Mulan PSL v2 at:
+
+	http://license.coscl.org.cn/MulanPSL2
+
+THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+See the Mulan PSL v2 for more details.
+*/
+package tenant
+
+import (
+	"sort"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/types"
+
+	cmdUtil "github.com/oceanbase/ob-operator/internal/cli/cmd/util"
+	"github.com/oceanbase/ob-operator/internal/cli/tenant"
+	"github.com/oceanbase/ob-operator/internal/clients"
+)
+
+// NewShowCmd show the overview and operations of ob tenant
+func NewShowCmd() *cobra.Command {
+	o := tenant.NewShowOptions()
+	logger := cmdUtil.GetDefaultLoggerInstance()
+	tbw, tbLog := cmdUtil.GetTableLoggerInstance()
+	cmd := &cobra.Command{
+		Use:     "show <tenant_name>",
+		Short:   "Show overview of an ob tenant",
+		Args:    cobra.ExactArgs(1),
+		PreRunE: o.Parse,
+		Run: func(cmd *cobra.Command, args []string) {
+			obtenant, err := clients.GetOBTenant(cmd.Context(), types.NamespacedName{
+				Namespace: o.Namespace,
+				Name:      o.Name,
+			})
+			if err != nil {
+				logger.Fatalln(err)
+			}
+			obtenantOperation, err := clients.GetOBTenantOperations(cmd.Context(), obtenant)
```

Comment:
Had better name a variable containing many entities to plural format or `xxxList` to get more semantical meaning.